docs/dataset/object-detection.md

```diff
@@ -0,0 +1,352 @@
+
+    ```shell
+    poetry run python3 object_detection_2d/upload_dataset.py
+    ```
+
+    After this script has completed, a new dataset named `coco-2014-val` will be created
+    and seen in [:kolena-dataset-20: Datasets](https://app.kolena.com/redirect/datasets).
+
+### Step 3: Format your Object Detection Model Results
+
+An object detection model will predict the bounding boxes of objects within each image, so there is a list of
+inferences per image:
+[`List[ScoredLabeledBoundingBox]`](../reference/annotation.md#kolena.annotation.ScoredLabeledBoundingBox).
+Note that a [`ScoredLabeledBoundingBox`](../reference/annotation.md#kolena.annotation.ScoredLabeledBoundingBox)
+has an extra `score` attribute for the confidence of the model:
+```
+{""top_left"": ... ""label"": ""car"", ""score"": 0.94, ""data_type"":""ANNOTATION/BOUNDING_BOX""}
+```
+
+The expected CSV format for model results should look like the CSV below:
+```
+locator,count_TP,count_FP,count_FN,TP,FP,FN,custom_information
+s3://my-images/intersection_1.png,2,0,0,"[{""top_left"": [538.5, 9.0], ... }, {""top_left"": ... }]","[]","[]",0.88
+s3://my-images/intersection_2.png,0,0,1,"[]","[]","[{""top_left"": ... }]",0.5
+...
+```
+
+By using a [`locator`](../dataset/advanced-usage/dataset-formatting/computer-vision.md#using-the-locator)
+as a unique reference to an image in the dataset, Kolena can join the model results in the CSV to the correct
+datapoint.
+
+See the
+[`object_detection_2d/upload_results.py`](https://github.com/kolenaIO/kolena/blob/trunk/examples/dataset/object_detection_2d/object_detection_2d/upload_results.py)
+example script for details on how results were generated.
+
+### Step 3: Upload Object Detection Model Results
+
+For this example, we will upload object detection model results
+of [YOLO X](https://github.com/Megvii-BaseDetection/YOLOX) for the single class object detection task in the web app,
+and upload the results of [Faster R-CNN](https://github.com/facebookresearch/Detectron) for the
+multiclass object detection task through the SDK.
+
+=== "Web App"
+
+    To upload new model results, from the `Details` tab of the dataset, click on `Upload Model Results`
+    in the upper right.
+    Then, select `Upload From Cloud Storage`. Using the explorer, navigate to
+    `s3://kolena-public-examples/coco-2014-val/person-detection/results/` and select the `yolo_x` CSV.
+
+    You will now see a preview of how Kolena will ingest the model results. Give your model a name,
+    and click `Import` to upload the model results.
+
+    <figure markdown>
+    ![Model Results Upload](../assets/images/task-od-upload-results-light.gif#only-light)
+    ![Model Results Upload](../assets/images/task-od-upload-results-dark.gif#only-dark)
+    <figcaption markdown>Model Results Upload</figcaption>
+    </figure>
+
+    It's also important to note down the evaluation configuration of the model being uploaded, for example:
+    `iou_threshold: 0.5` or `iou_threshold: 0.3`.
```

Comment:
It feels like this part isn't clear enough -- we should either drop it for brevity, or flesh it out (in this example, there should be an eval config with `iou_threshold`, `min_confidence_score`, and `threshold_strategy` -- we don't really touch on them outside of the short blurb you have for `iou_threshold`)